src/gitea/pull_request/async_pull_request.py

```
@@ -0,0 +1,120 @@
+
+from aiohttp import ClientResponse
+
+from gitea.pull_request.base import BasePullRequest
+from gitea.resource.async_resource import AsyncResource
+from gitea.utils.response import process_async_response
+
+
+class AsyncPullRequest(BasePullRequest, AsyncResource):
+    """Asynchronous Pull Request API for Gitea."""
+
+    async def _list_pull_requests(  # noqa: PLR0913
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)